hack/update-csv-timestamp.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
I saw the issue where this was mentioned, in our case this will render w/ d/s bundle date not being updated.

Maybe we need to settle w/ manual backports if CSV is updated and separate that into an individual commit so that other commits part of same PR can be cherry-picked and bundle can be generated.

How about we check bundle generation diff w/ removing this field as done in https://github.com/red-hat-storage/odf-operator/commit/3e4f3eacff049a727348c6592a4b32d3698ae10c?

In addition to work done in #194, above does the same for bundle? wdyt?

Reply:
yes we can do that